guides/common/modules/proc_content-view-steps.adoc

```diff
@@ -0,0 +1,19 @@
+[id="Creating_a_Content_View_{context}"]
```

Comment:
Please change the file name to `proc_creating-a-content-view.adoc` (and alway in the assembly).

Reply:
@maximiliankolb I just realized that I was basically duplicating an already existing section for this. There already is a file name called `proc_creating-a-content-view.adoc`. I went ahead and updated the steps to reflect these changes in that file while also updating the `proc_creating-a-content-view-short.adoc` to get rid of the mention of Component Content View. You will see these changes when I commit again. The assembly has been updated as well with the removal of the file I had initially created.